include/micm/CPU.hpp

```diff
@@ -11,37 +11,27 @@
 
 namespace micm
 {
-  using DenseMatrixVector = micm::VectorMatrix<double, MICM_DEFAULT_VECTOR_SIZE>;
-  using SparseMatrixVector = micm::SparseMatrix<double, micm::SparseMatrixVectorOrdering<MICM_DEFAULT_VECTOR_SIZE>>;
-
-  using DenseMatrixStandard = micm::Matrix<double>;
-  using SparseMatrixStandard = micm::SparseMatrix<double, micm::SparseMatrixStandardOrdering>;
-
-  using VectorState = micm::State<DenseMatrixVector, SparseMatrixVector>;
-  using StandardState = micm::State<DenseMatrixStandard, SparseMatrixStandard>;
-
-  using RosenbrockVectorType = typename micm::RosenbrockSolverParameters::
-      template SolverType<micm::ProcessSet, micm::LinearSolver<SparseMatrixVector, micm::LuDecomposition>>;
-  using Rosenbrock = micm::Solver<RosenbrockVectorType, micm::State<DenseMatrixVector, SparseMatrixVector>>;
-
-  using RosenbrockStandardType = typename micm::RosenbrockSolverParameters::
-      template SolverType<micm::ProcessSet, micm::LinearSolver<SparseMatrixStandard, micm::LuDecomposition>>;
-  using RosenbrockStandard = micm::Solver<RosenbrockStandardType, micm::State<DenseMatrixStandard, SparseMatrixStandard>>;
-
-  using BackwardEulerVectorType = typename micm::BackwardEulerSolverParameters::
-      template SolverType<micm::ProcessSet, micm::LinearSolver<SparseMatrixVector, micm::LuDecomposition>>;
-  using BackwardEuler = micm::Solver<BackwardEulerVectorType, micm::State<DenseMatrixVector, SparseMatrixVector>>;
-
-  using BackwardEulerStandardType = typename micm::BackwardEulerSolverParameters::
-      template SolverType<micm::ProcessSet, micm::LinearSolver<SparseMatrixStandard, micm::LuDecomposition>>;
-  using BackwardEulerStandard =
-      micm::Solver<BackwardEulerStandardType, micm::State<DenseMatrixStandard, SparseMatrixStandard>>;
-
-  using RosenbrockThreeStageBuilder =
-      micm::CpuSolverBuilder<micm::RosenbrockSolverParameters, DenseMatrixVector, SparseMatrixVector>;
-  using BackwardEulerBuilder = micm::CpuSolverBuilder<
-      micm::BackwardEulerSolverParameters,
-      DenseMatrixVector,
-      SparseMatrixVector,
-      micm::LuDecompositionDoolittle>;
+  using DenseMatrixVector = VectorMatrix<double, MICM_DEFAULT_VECTOR_SIZE>;
```

Comment:
Is there a reason to remove `micm::` here?

Reply:
It's already inside of the `micm` namespace

Reply:
Thanks for the clarification.